### PR TITLE
CIS update blog post (rebased)

### PR DESCRIPTION
**Pull request by sej7278**
squashed the cis blog post commits into one from #361 as a single new markdown file shouldn't need 6 commits :-)

changed date again and renamed accordingly

used a non-master branch

@bennyvasquez FYI

### CODE REVIEW

File: content/blog/2023-11-28-cis-benchmarks-update.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: "CIS Benchmarks Update"
+type: blog
+author:
+ name: "Simon John"
+ bio: "Security Standards Architect"
+ image: /users/sjohn.png
+date: '2023-11-28'
+post:
```
Comment (bennyvasquez) [inline content/blog/2023-11-28-cis-benchmarks-update.md:9]:
```suggestion
images:
  - /blog-images/cis_update.svg
post:
```

Comment (bennyvasquez) [reply content/blog/2023-11-28-cis-benchmarks-update.md:9]: Added this from our updated template. This is the image that gets shared when the link is posted to social media.

**Comment by bennyvasquez** [comment]
Other than that one thing, I think we're good to go on this!

File: content/blog/2023-11-28-cis-benchmarks-update.md (hunk 1 above)
Comment (sej7278) [reply content/blog/2023-11-28-cis-benchmarks-update.md:9]: Ah right I wondered why we got the anaconda image all the time on LinkedIn etc

**Comment by bennyvasquez** [comment]
I'll merge next week when we publish it.

**Review by bennyvasquez — APPROVED** [review]

**Comment by sej7278** [comment]
i'm not convinced the new social image thing works though - linkedin shows my bio picture and twitter shows nothing (or a broken image?)

**Comment by bennyvasquez** [comment]
it works overall, as you can see from this post: https://www.linkedin.com/feed/update/urn:li:activity:7132789516783742976 

The SVG thing might be a holdup. Want to submit a new PR with a png?

**Comment by sej7278** [comment]
@bennyvasquez sure see https://github.com/AlmaLinux/almalinux.org/pull/401

**Pull request merged**